evm/verify-token-deployer.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+const axios = require('axios');
+const { Command, Option } = require('commander');
+const { ethers } = require('hardhat');
+const {
+    utils: { defaultAbiCoder, isHexString, keccak256, toUtf8Bytes },
+} = ethers;
+
+const { loadConfig, printError } = require('./utils');
+
+async function processCommand(chain, options) {
+    try {
+        const { signedHash, env, address, message, api } = options;
+        const { txHash } = options;
+        const eventHash = keccak256(toUtf8Bytes('InterchainTokenDeployed(bytes32,address,address,string,string,uint8)'));
```

Comment:
Ideally you should get the event signature from the ITS abi which can be gotten via `getContractJSON('InterchainTokenService')` rather than typing it out (prone to typos, not update proof, etc)